frontend/src/components/AdminControls/Recipes.css

```diff
@@ -0,0 +1,88 @@
+.react-bootstrap-table-pagination-list {
+  text-align: center;
+  transform: translateX(-50%);
+}
+
+.react-bootstrap-table-page-btns-ul {
+  float: none;
+}
+
+.react-bootstrap-table-pagination-list {
+  text-align: center;
+  transform: translateX(-50%);
+}
+
+.react-bootstrap-table-page-btns-ul {
+  float: none;
+}
+
+.button {
```

Comment:
I'm a little concerned about these very generic class names like `button`, `line`, `container`, `header`, etc. The CSS is at least partially shared across the site, so things defined in one file can overwrite things defined in another. It might be good to use more specific classes where the behavior is specific to the page, like `.adm-pg-button`, or the like. I know `container` and `header`, for example, are used on other pages.

Right now, the overlap is minimal because no other pages are using reactstrap. But as that changes, things might start getting.. unpleasant.